components/chart/labels-template-and-format.md

```diff
@@ -10,15 +10,9 @@ position: 22
 
 # Label Template and Format
 
-The Chart for Blazor can render labels on the axes, series data points and legend. You can control those texts not only through the values you data bind, but also through [format strings](#format-strings) or [templates](#templates).
+The Chart for Blazor can render labels on the axes, series data points and legend. You can control those labels through the values in the Chart `Data`, but also through [format strings](#format-strings) and [templates](#templates).
 
-To turn on series labels, you must set their `Visible` property to `true` under the corresponding `ChartSeriesLabels` tag. The series labels are turned off by default to avoid clutter and to make the charts easier to read.
-
-You can jump to the following sections in this article:
-
-* [Format Strings](#format-strings)
-* [Templates](#templates)
-	* [Example: Hide Label Conditionally](#hide-label-conditionally)
+To turn on series labels, you must set their `Visible` property to `true` under the corresponding `ChartSeriesLabels` tag. The series labels are turned off by default to avoid clutter and to make the Chart easier to read.
```

Comment:
```suggestion
To turn on series labels, set their `Visible` property to `true` under the corresponding `ChartSeriesLabels` tag. The series labels are turned off by default to avoid clutter and to make the Chart easier to read.
```